crates/holochain/tests/tests/multi_conductor/mod.rs

```diff
@@ -231,6 +231,16 @@ async fn private_entries_dont_leak() {
     let apps = conductors.setup_app("app", &dnas).await.unwrap();
     let ((alice,), (bobbo,)) = apps.into_tuples();
 
+    conductors[0]
+        .require_initial_gossip_activity_for_cell(&alice, 1, std::time::Duration::from_secs(30))
```

Comment:
party time

Funny that no one had written this function before.
Or rather typical.